genesis/process/metaGenesisBlockCreator.go

```diff
@@ -381,8 +381,8 @@ func createProcessorsForMetaGenesisBlock(arg ArgsGenesisBlockCreator, enableEpoc
 		ScProcessor:                           scProcessor,
 		TxTypeHandler:                         txTypeHandler,
 		EconomicsFee:                          genesisFeeHandler,
-		ESDTEnableEpoch:                       arg.EpochConfig.EnableEpochs.ESDTEnableEpoch,
-		BuiltInFunctionOnMetachainEnableEpoch: arg.EpochConfig.EnableEpochs.BuiltInFunctionOnMetaEnableEpoch,
+		ESDTEnableEpoch:                       enableEpochs.ESDTEnableEpoch,
```

Comment:
we should follow @bogdan-rosianu suggestion an refactor these config passing using a specialized struct. This will be a new PR